spring-integration-redis/src/main/java/org/springframework/integration/redis/outbound/RedisQueueOutboundChannelAdapter.java

```diff
@@ -77,6 +80,10 @@ public void setSerializer(RedisSerializer<?> serializer) {
 		this.serializerExplicitlySet = true;
 	}
 
+	public void setLeftPush(boolean leftPush) {
```

Comment:
Hello, @rainerfrey !

Thank you for the quick fix!

Please, receive the next review phase.

This method must be with JavaDocs as well.